### PR TITLE
Enable CPU and Memory accounting on vagrant cluster

### PR DESCRIPTION
**Pull request by derekwaynecarr**
Only the last commit is relevant for this PR.

### CODE REVIEW

**Comment by derekwaynecarr** [comment]
@eparis - what is the best way we could get this drop-in file added when doing RPM installs?

https://github.com/kubernetes/kubernetes/pull/26174/files#diff-9d46a3e18330573ebc5ef0b15a593409R20

**Comment by derekwaynecarr** [comment]
@yifan-gu - I will also update the systemd node spec, but here is the PR I discussed yesterday that enables global accounting by default.

**Comment by derekwaynecarr** [comment]
Needs a little more testing

File: cluster/vagrant/provision-utils.sh [hunk 1]
```diff
@@ -14,6 +14,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+function enable-accounting() {
+  echo "Enable accounting"
```
Comment (ihmccreery) [inline cluster/vagrant/provision-utils.sh]: "Enabling accounting"?

**Comment by derekwaynecarr** [comment]
chatted with @eparis @ingvagabund - will add the same file in https://github.com/kubernetes/contrib/tree/master/init/systemd so RPM based install will pick it up as well.

**Comment by derekwaynecarr** [comment]
@ihmccreery - comment addressed.

**Comment by ihmccreery** [comment]
@derekwaynecarr Sorry to thrash you on this, but I have zero context for this.  Is there a better reviewer for this PR than myself?

**Comment by derekwaynecarr** [comment]
@ihmccreery - no problem.  I am usually the best reviewer for this.

@ncdc - can you review my work?

File: cluster/vagrant/provision-utils.sh [hunk 2]
```diff
@@ -113,8 +119,10 @@ function install-salt() {
   popd
 
   if ! which salt-call >/dev/null 2>&1; then
-    # Install salt binaries
-    curl -sS -L --connect-timeout 20 --retry 6 --retry-delay 10 https://bootstrap.saltstack.com | sh -s
+    # Install salt from official repositories.
+    # Need to enable testing-repos to get version of salt with fix for dnf-core-plugins
+    dnf config-manager --set-enabled updates-testing
```
Comment (ncdc) [inline cluster/vagrant/provision-utils.sh]: Do you want to enable updates-testing forever here, or just do `dnf install --enablerepo=updates-testing -y salt-minion` below?

**Comment by ncdc** [comment]
@derekwaynecarr 1 question re updates-testing

**Comment by derekwaynecarr** [comment]
@ncdc Yes, I need it for all at the moment.  tagging lgtm.

**Comment by ncdc** [comment]
OK LGTM

**Comment by a-robinson** [comment]
@k8s-bot node e2e test this: #25812

**Pull request merged**